src/InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs

```diff
@@ -30,10 +30,7 @@ public override void Build(BatchComparisonResult comparisonResult, OSPSuiteTrack
             validationResultFor(comparisonResult)
          };
 
-         var fileComparisonResults = comparisonResult.FileComparisonResults.Where(x => x.State != ValidationState.Valid).ToList();
-         if (fileComparisonResults.Any())
-            objectsToReport.Add(new Paragraph(Assets.Reporting.FailedValidations));
-
+         var fileComparisonResults = comparisonResult.FileComparisonResults.OrderByDescending(x => x.State);
```

Comment:
That's good.
Can you also make sure that comparison results with the same `ValidationState `are sorted alphabetically by the `FileName `within their group?